pkg/aggregator/aggregator.go

```diff
@@ -212,6 +216,11 @@ type BufferedAggregator struct {
 	contLcycleStopper     chan struct{}
 	contLcycleDequeueOnce sync.Once
 
+	contImageIn          chan senderContainerImage
+	contImageBuffer      chan senderContainerImage
+	contImageStopper     chan struct{}
+	contImageDequeueOnce sync.Once
```

Comment:
It’s more the fact that we send data to a brand new Event-Platform track rather than the fact that it’s a new check which triggers all this plumbing.
I mainly copied what was done for the other Event-Platform tracks that the agent is feeding like `ContainerLifecycle` for ex.
I supposed that the use of different data types to be sent to different endpoints justified to have dedicated functions.